lib/gears/object.lua

```diff
@@ -59,6 +59,21 @@ function object:connect_signal(name, func)
     sig.strong[func] = true
 end
 
+--- Check if the callback is connected to the signal.
+--
+-- @tparam string name The name of the signal.
+-- @tparam function func The callback to check if connected.
+-- @treturn boolean Whether the signal is connected.
+-- @method is_signal_connected
```

Comment:
Note to myself : this doc should also be added to the `docs/common/object.ldoc` file

Reply:
done